src/app/loadout-builder/filter/FilterBuilds.tsx

```diff
@@ -49,17 +48,28 @@ export default function FilterBuilds({
           onStatFiltersChanged={onStatFiltersChanged}
           onStatOrderChanged={onStatOrderChanged}
         />
-        <div
-          className={styles.assumeMasterwork}
-          title={t('LoadoutBuilder.AssumeMasterworkDetailed')}
-        >
-          <CheckButton
-            name="lo-assume-masterwork"
-            checked={assumeMasterwork}
-            onChange={(checked) => dispatch(setSetting('loAssumeMasterwork', checked))}
+        <div className={styles.upgradeMaterials}>
+          <select
+            className={styles.materialSelect}
+            value={upgradeSpendTier}
+            onChange={(event) =>
+              dispatch(setSetting('loUpgradeSpendTier', parseInt(event.target.value, 10)))
+            }
           >
-            {t('LoadoutBuilder.AssumeMasterwork')}
-          </CheckButton>
+            <option value={UpgradeSpendTier.Nothing}>{t('LoadoutBuilder.SpendNoMaterials')}</option>
```

Comment:
Bonus points for injecting the localized name of the material from manifest.

Bonus bonus points for using our fancy Select to show a picture of the material next to the name.

Reply:
Yeah I can do this but it probably won't be until Sunday night. I didn't realise that fancy select existed. Thoughts on adding an unselectable option to it where I can explain what the options mean?